src/python/pants/backend/jvm/tasks/resolve_shared.py

```diff
@@ -65,3 +74,17 @@ def add_directory_digests_for_jars(self, targets_and_jars):
       snapshotted_targets_and_jars.append((target, snapshotted_jars))
 
     return snapshotted_targets_and_jars
+
+  @abstractproperty
+  def resolver_name(self):
+    """???"""
```

Comment:
Add a docstring or remove this docstring, but ??? isn't a useful docstring... (and below)

Reply:
Fixed!